### ISSUE TITLE
How to get the Link component to work?

### ISSUE THREAD

**Issue by jwag59** [bigeyex/python-adminui#46]
I have used the Link component on a page but can't seem to get the Link to work.
I have tried both absolute and relative specifications in the 'link_to=' parameter but nothing happens when you click on the link.

```Link('View Nodegroups', link_to='http://10.140.9.50:8080/ngrps'),```

and

```Link('View Nodegroups', link_to='/ngrps'),```

And when I inspect the web page the following element is shown for this component (i.e. there is no 'href' option specified):
```<a>View Nodegroups</a>```

**Comment by harry3689** [comment]
I believe you have to use the on_click parameter of the Link component which should return the NavigateTo component:

```python
@app.page("/nodegroups", "NodeGroups")
def node_groups():
    return [Card(content=[Link("View NodeGroups", on_click=NavGroup)])]

def NavGroup():
   return [NavigateTo("http://10.140.9.50:8080/ngrps")]
```

Now, if you click on the Link it should redirect you to the desired URL.

**Comment by harry3689** [comment]
Yes your analysis does make sense. I agree that the implementation can be improved.

**Comment by jwag59** [comment]
In adminui version 1.4.3 the 'link_to' URL parameter seems to have magically started to work as I expected. So you no longer need to code the on_click option. This is great.

**Comment by bigeyex** [comment]
There is no magic. I fixed a bug in 1.4.4.
